docs/docs/how-to/profile_selectors.md

```diff
@@ -45,6 +48,8 @@ Let's break down the example above:
   - The first selector filters repositories to include only those that are not forks and whose name starts with stacklok. In other words, those that are part of the stacklok organization.
   - The second selector filters artifacts to include only those provided by `github-app-stacklok`.
   - The third selector filters repositories to include only those with a GPL license and the fourth selector filters repositories to include only those written in Go. These two selectors use the properties map which is provider-specific.
+  - The fourth selector filters repositories to include only that use Go as the primary language.
+  - The fifth selector filters repositories to include only those provided by the GitHub provider.
```

Comment:
it's a bit embarassing but we already had documentation on the provider fields. So the only docs I added are examples.